README.rst

```diff
@@ -77,7 +77,7 @@ Here is an Example migration file::
         backup: # Defines how the backup should be done before the migration.
           command: echo "backup command on ${DB_NAME}"
           stop_on_failure: true
-          ignore_if: test "${RUNNING_ENV}" != "prod"
+          ignore_if: test "${RUNNING_ENV}" != "full"
```

Comment:
:warning: RUNNING_ENV could be dev,test,int or prod IIRC.

It's MARABUNTA_MODE that is either full or sample.